homeassistant/components/volumio/media_player.py

```diff
@@ -36,7 +39,7 @@
 from .browse_media import browse_node, browse_top_level
 from .const import DATA_INFO, DATA_VOLUMIO, DOMAIN
 
-_CONFIGURING = {}
+_CONFIGURING: dict[str, Any] = {}
```

Comment:
I'm not even sure what this is for. It's been there for over 4 years (since this integration was introduced, and I don't think it's used anywhere.

Reply:
Looking at other media_player implementations, many have it, some use it. So I believe it is safe to remove it.
Thanks.